sandbox.html

```diff
@@ -611,6 +618,42 @@
   </div>
 </template>
 
+<template data-action="fetchAreaIntersection" data-common="layer">
+  <div>
+    <label for="embeddingTargetId">Embedding target ID *</label>
+    <input name="embeddingTargetId" id="embeddingTargetId" required>
+  </div>
+  <div>
+    <label for="geometry">Intersection Geometry (GeoJSON) *</label>
+    <textarea name="geometry" id="geometry" rows="5" required></textarea>
+    <small>
+      <p>
+        The geometry must result in an area in combination with the buffer specification.
+      </p>
+    </small>
+  </div>
+  <div>
+    <label>
+      <input type="checkbox" name="useMapSrs">
+      Use map SRS
+    </label>
+  </div>
+  <div>
+    <label for="distance">Buffer size</label>
+    <input name="distance" id="distance">
+    <label for="lengthUnit">Length unit</label>
+    <select name="lengthUnit" id="lengthUnit">
+      <option>m</option>
+      <option>km</option>
+    </select>
+    <small>
+      <p>
+        A positive buffer size is needed for point or line intersection geometries.
```

Comment:
Should we add an assertion for the distance to be positive?

Reply:
polygons can have a negative distance

Reply:
Yeah, but why not? We have that info in the method.